.gitignore

```diff
@@ -1,3 +1,4 @@
+.vscode
```

Comment:
Sorry I think I didn't explain this clearly. If you are using `vscode`, you can add this to `.gitignore` so that the changes in `.vscode` won't be committed. But you need to remove this from `.gitignore` before creating a PR. However, IDEs such as `IntelliJ IDEA` are still highly recommended instead of `vscode`.